journey-map/public/index.html

```diff
@@ -34,6 +34,45 @@ <h1>
         </div>
       </div>
       <div id="header-content">
+        <form id="transitions-traffic-form">
+            <h2>Transitions Traffic</h2>
```

Comment:
```suggestion
            <h2>User Traffic</h2>
```
Might be better for non-technicals?